cli/main.rs

```diff
@@ -33,10 +33,10 @@ fn main() {
     );
 
     let mut options = jstime::Options::default();
-    options.snapshot = Some(include_bytes!(concat!(
-        env!("OUT_DIR"),
-        "/snapshot_data.blob"
-    )));
+    // options.snapshot = Some(include_bytes!(concat!(
+    //     env!("OUT_DIR"),
+    //     "/snapshot_data.blob"
+    // )));
```

Comment:
disabling snapshots here so that we can at least see things working. Re-Enabling snapshots makes URL no longer exist on globalThis